Admin/CONTRIBUTION.md

```diff
@@ -0,0 +1,41 @@
+# Contribution
+Author: Matthew Harper
+
+> [!NOTE]
+> Information here is likely to change year-by-year. Please check in with the current CCDC Coaches and Captains.
+---
+This repository is used by the University of Massachusetts Lowell (student) Cyber Security Teams. It contains scripts, documentation, notes and some scenarios. Anything that should be kept confidential or private should **NOT** be pushed to this repository. Anything that is pushed to this is public and can be seen by anyone so passwords, usernames, etc are often temporary or placeholders used in the initial infrastructure deployments.
+
+## Keep in mind
+> [!IMPORTANT]
+> ***Do Not*** Push Personal Information.
+
+1. All documents should have all authors listed at the top of scripts and documents.
+    ```
+    Author: XYZ
+    Author: ZYX (Updated 2024)
+    ...
+    ```
+2. All scripts should be working and have the execute permission bit set!
+3. All documents should be complete and somewhat well written.
+4. Any usernames and passwords should be placeholders if they are included as part of the commit.
+
+## Guidelines and Procedure
+> [!IMPORTANT]
+> If you make a branch it should be *focused* that is we should keep the changes in that branch to specific technology or set of files. You should not be updating Windows, AWS and Linux documents unless you are working with a technology that uses all three of those things. In that case it may be a good idea for the technology (such as Wazuh) to be in it's own folder.
```

Comment:
Can you a outline a naming convention for branches?

Reply:
Sure